master/internal/config/saml_config.go

```diff
@@ -17,8 +17,8 @@ type SAMLConfig struct {
 	AutoProvisionUsers          bool   `json:"auto_provision_users"`
 	GroupsAttributeName         string `json:"groups_attribute_name"`
 	DisplayNameAttributeName    string `json:"display_name_attribute_name"`
-	AgentUIDAttributeName       int    `json:"agent_uid_attribute_name"`
-	AgentGIDAttributeName       int    `json:"agent_gid_attribute_name"`
+	AgentUIDAttributeName       string `json:"agent_uid_attribute_name"`
+	AgentGIDAttributeName       string `json:"agent_gid_attribute_name"`
```

Comment:
Being an "attribute name" I think string makes more sense here.

Reply:
That's a fair point! I wonder why it was this way to begin with?

Reply:
Oh it was because it wasn't originally the attribute names? It was the values, and we later shifted that

Reply:
Yes, exactly!